manifests/0000_70_cluster-network-operator_03_deployment.yaml

```diff
@@ -21,12 +21,15 @@ spec:
         image: quay.io/openshift/origin-cluster-network-operator:latest
         command:
         - "/usr/bin/cluster-network-operator"
-        - "--url-only-kubeconfig=/etc/kubernetes/kubeconfig"
+        - "start"
+        - "--listen=0.0.0.0:9104"
```

Comment:
so this is for metrics? We weren't exposing them before?

Reply:
correct, the CNO was not exposing any metrics.

We don't yet expose any super useful ones, but this sets us up to expose them. And we don't need kube-rbac-proxy even.